arduino-core/src/processing/app/i18n/Resources_en.po

```diff
@@ -116,6 +116,16 @@ msgstr "A new version of Arduino is available,\nwould you like to visit the Ardu
 msgid "A newer {0} package is available"
 msgstr "A newer {0} package is available"
 
+#: ../../../../../arduino-core/src/cc/arduino/contributions/packages/ContributionsIndexer.java:97
+msgid ""
+"A package index has an invalid signature and needs to be updated.\n"
+"Please open the Board Manager from the menu\n"
+"\n"
+"      Tools -> Board -> Board Manager\n"
```

Comment:
```suggestion
"Please open Boards Manager from the menu\n"
"\n"
"      Tools > Board > Boards Manager\n"
```
- Removal of "the" from "the Board Manager". I consider "Boards Manager" to be a proper noun, thus an article is not appropriate. However, this is very much a subjective thing and I'm not heavily invested in my opinion, so leaving the "the" is OK too.
- The feature is named "Boards Manager", not "Board Manager".
- The most common convention for indicating menu paths in the IDE's messages and Arduino's documentation is ">", rather than "->"